package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@node-loader/core": "2.0.0",
     "chalk": "4.1.2",
+    "classnames": "^2.3.1",
```

Comment:
this got added in the wrong place
for poker i'm seeing
```
[vite] Internal server error: Failed to resolve import "classnames" from "../../client/web/src/components/Lobby.tsx". Does the file exist?
```

Reply:
ahh opps, let me move it the the right place